docs/src/main/sphinx/optimizer/cost-based-optimizations.rst

```diff
@@ -73,14 +76,40 @@ The valid values are:
 Capping replicated table size
 -----------------------------
 
-Join distribution type will be chosen automatically when join reordering strategy
-is set to ``AUTOMATIC`` or when join distribution type is set to ``AUTOMATIC``.
-In such case it is possible to cap the maximum size of replicated table via
-``join-max-broadcast-table-size`` config property (e.g. ``join-max-broadcast-table-size=100MB``)
-or via ``join_max_broadcast_table_size`` session property (e.g. ``set session join_max_broadcast_table_size='100MB';``)
-This allows to improve cluster concurrency and to prevent bad plans when CBO misestimates size of joined tables.
+Join distribution type is automatically chosen when join reordering strategy is
+set to ``AUTOMATIC`` or when join distribution type is set to ``AUTOMATIC``. In
+such cases, it is possible to cap the maximum size of the replicated table via
+the ``join-max-broadcast-table-size`` configuration property (such as
+``join-max-broadcast-table-size=100MB``) or via the
+``join_max_broadcast_table_size`` session property (such as
+``set session join_max_broadcast_table_size='100MB';``). This allows you to
+improve cluster concurrency and prevent bad plans when CBO misestimates size
```

Comment:
when the cost-based optimizer misestimates the size of the joined tables.

... also .. is that technically correct .. the optimizer does NOT do any estimation I think .. it just uses the available table stats .. or it doesnt